submarine-security/spark-security/pom.xml

```diff
@@ -539,6 +563,7 @@
       <id>spark-2.3</id>
       <properties>
         <spark.version>2.3.4</spark.version>
+        <guava.version>15.0</guava.version>
```

Comment:
can you explain why we need to add this?

Reply:
this change for test case in profile  hadoop-3.x spark-3.x, case runner has guava error 
`java.lang.NoSuchMethodError: com.google.common.base.Preconditions.checkArgument(ZLjava/lang/String;Ljava/lang/Object;)V`
Dependency guava version is old; 
so, i add guava Dependency for test scope, profile `hadoop-3.2, ranger-x, spark-3.0 ` test case passed, 
but profile `hadoop-2.9, ranger-x, spark-2.x` test case is failure [Cause: java.lang.IllegalAccessError: tried to access method com.google.common.base.Stopwatch.<init>](https://stackoverflow.com/questions/36427291/illegalaccesserror-to-guavas-stopwatch-from-org-apache-hadoop-mapreduce-lib-inp) , spark-2.x need guava version is 15.0;

if skip profile `hadoop-3.x` test case, i well be clean it